### ISSUE TITLE
Web component: LightboxMedia Viewer - focus within the modal dialog

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4057]
## Detailed description
On lightbox media viewer - default &  Embedded video player
Issue: 
Actual Result:-
while navigating using tab/shift tab, focus is not trapped with in the modal dialog.

Expected Result:
Focus should be trapped with in the dialog.
